Tasks/Fedorchuk/task5/task5.py

```diff
@@ -0,0 +1,51 @@
+def check_str():
+    word ="The quick Brow Fox"
+    upper=""
+    lower="" 
+    for i in word:
+        if i.isupper() == True:
+            upper+=i
+            value1=len(upper)
+        if i.islower()==True:
+            lower+=i
+            value2=len(lower)
```

Comment:
the value1 and value2 calculation could be done at the end of the cycle, and not inside